dbms/src/Flash/Mpp/MPPTunnel.h

```diff
@@ -256,16 +258,20 @@ class LocalTunnelSender : public TunnelSender
         LocalRequestHandler & local_request_handler_,
         const LoggerPtr & log_,
         MemoryTrackerPtr & memory_tracker_,
-        const String & tunnel_id_)
+        const String & tunnel_id_,
+        std::function<void()> && add_local_conn_num)
         : TunnelSender(memory_tracker_, log_, tunnel_id_, nullptr)
         , source_index(source_index_)
         , local_request_handler(local_request_handler_)
         , is_done(false)
-    {}
+    {
+        add_local_conn_num();
```

Comment:
how about `connect`

Reply:
> how about `connect`

The purpose of this function is to modify the variable in `ExchangeReceiver`. I think the name `connect` can't show this purpose.